reference/sql/statements/create-sequence.md

```diff
@@ -0,0 +1,322 @@
+---
+title: CREATE SEQUENCE
+summary: An overview of the usage of CREATE SEQUENCE for the TiDB database.
+category: reference
+---
+
+# CREATE SEQUENCE
+
+The `CREATE SEQUENCE` statement creates sequence objects in TiDB. The sequence is a database object that is on a par with the table and the `View` object. The sequence is used to generate serialized IDs in a customized way.
+
+## Synopsis
+
+**CreateSequenceStmt:**
+
+![CreateSequenceStmt](/media/sqlgram-dev/CreateSequenceStmt.png)
+
+**OptTemporary:**
+
+![OptTemporary](/media/sqlgram-dev/OptTemporary.png)
+
+**IfNotExists:**
+
+![IfNotExists](/media/sqlgram-dev/IfNotExists.png)
+
+**TableName:**
+
+![TableName](/media/sqlgram-dev/TableName.png)
+
+**CreateSequenceOptionListOpt:**
+
+![CreateSequenceOptionListOpt](/media/sqlgram-dev/CreateSequenceOptionListOpt.png)
+
+**SequenceOption:**
+
+![SequenceOption](/media/sqlgram-dev/SequenceOption.png)
+
+**CreateTableOptionListOpt:**
+
+![CreateTableOptionListOpt](/media/sqlgram-dev/CreateTableOptionListOpt.png)
+
+## Syntax
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE [TEMPORARY] SEQUENCE [IF NOT EXISTS] sequence_name
+    [ INCREMENT [ BY | = ] increment ]
+    [ MINVALUE [=] minvalue | NO MINVALUE | NOMINVALUE ]
+    [ MAXVALUE [=] maxvalue | NO MAXVALUE | NOMAXVALUE ]
+    [ START [ WITH | = ] start ]
+    [ CACHE [=] cache | NOCACHE | NO CACHE]
+    [ CYCLE | NOCYCLE | NO CYCLE]
+    [ ORDER | NOORDER | NO ORDER]
+    [table_options]
+```
+
+## Parameters
+
+|Parameters | Default value | Description |
+| :-- | :-- | :--|
+| `TEMPORARY` | `false` | TiDB currently does not support the `TEMPORARY` option and provides only syntax compatibility for it. |
+| `INCREMENT` | `1` | The increment to use for values. Its positive or negative values can control the growth direction of the sequence. |
```

Comment:
Because this feature is borrowed from MariaDB, I've checked the [MariaDB document](https://mariadb.com/kb/en/create-sequence/) and referred to their language which might be a little different from the original docs-cn PR. BTW, I am not sure whether `step size` is correct here. @AilinKid

Reply:
Yes, Mariadb use `increment` to describe it directly,  so I think both of these are ok.

Reply:
Then maybe "Specifies the increment of a sequence" or "Specifies the step size of a sequence"? @TomShawn

Reply:
OK